vtex/faststore
# refactor: Import `SlideOver` component from the starter

## What's the purpose of this pull request?

Extract the `SlideOver` component from our starters into `@faststore/ui`.

## How to test it?

Run `yarn storybook` and check the component's page at http://localhost:6006/?path=/docs/molecules-slideover--default.

![CleanShot 2022-08-30 at 14 05 44](https://user-images.githubusercontent.com/381395/187497586-3e9fa674-c571-474e-9c17-36c5487d090d.gif)

### Starters Deploy Preview

- https://github.com/vtex-sites/nextjs.store/pull/225
